src/metabase/task/follow_up_emails.clj

```diff
@@ -120,7 +120,7 @@
     (when (should-send-abandoment-email?)
```

Comment:
If there's no such thing as an abandonment email anymore, we shouldn't have a `should-send-abandoment-email?` predicate function anymore.

Reply:
Also... I'm a little confused as to why you would leave the function name of `send-abandoment-email-if-needed!` itself even tho you changed it to send a normal follow-up email